tests/framework/cli/micropkg/test_micropkg_requirements.py

```diff
@@ -263,4 +264,7 @@ def test_complex_requirements(
             obj=fake_metadata,
         )
         assert result.exit_code == 1
-        assert "InvalidRequirement: Parse error" in result.output
+        assert (
+            "InvalidRequirement: Expected package name at the start of dependency specifier"
+            or "InvalidRequirement: Expected end or semicolon" in result.output
+        )
```

Comment:
I might be wrong here, but isn't this an equivalent of

```python
assert "a" or "b" in str
```

which evaluates as

```python
assert "a" or ("b" in str)
```

when we want something like

```python
assert ("a" in str) or ("b" in str)
```

? Just tested it:

```
>>> "a" or "b" in "c"
'a'
>>> "a" in "c" or "b" in "c"
False
>>> ("a" in "c") or ("b" in "c")
False
```

Reply:
Omg, yes you're so right! I shouldn't be doing PRs on Friday afternoon 🤦‍♀️ 😅